spec/unit/promise_spec.rb

```diff
@@ -747,6 +826,32 @@
         expect(result).to eq(reason)
         expect(subject.reason).to eq(reason)
       end
+
+      it 'allows specifying a list of error types' do
+        result = nil
+        p = subject.catch(StandardError) { |reas| result = reas }
+
+        subject.reject(reason)
+        expect(result).to eq(reason)
+        expect(subject.reason).to eq(reason)
+        expect(p).to be_fulfilled
+
+        result = nil
+        p = subject.catch(RuntimeError) { |reas| result = reas }
+
+        subject.reject(reason)
+        expect(result).to be_nil
+        expect(subject.reason).to eq(reason)
+        expect(p).to be_rejected
+
+        result = nil
+        p = subject.catch(RuntimeError, StandardError) { |reas| result = reas }
+
+        subject.reject(reason)
+        expect(result).to eq(reason)
+        expect(subject.reason).to eq(reason)
+        expect(p).to be_fulfilled
+      end
```

Comment:
I don't think these tests for `catch` are necessary, since it is just an alias for `rescue`